package.json

```diff
@@ -98,6 +98,7 @@
     "@angular/compiler": "^8.0.0",
     "@angular/core": "^8.0.0",
     "@angular/forms": "^8.0.0",
+    "@angular/http":"^7.2.16",
```

Comment:
Can you run the `yarn` command? We've recently made the change to not update the `yarn.lock` file whenever contributors run Oppia. As a consequence, when a contributor does make a change to `package.json`, they will have to make sure that they properly update the `yarn.lock` file. 

/cc @vojtechjelinek 

/cc @ankita240796 this can be a new Oppiabot check. Specifically, whenever `package.json` is changed, `yarn.lock` must be accompanied (but not vice versa).

Reply:
Created an issue to keep track: https://github.com/oppia/oppiabot/issues/77